### DISCUSSION TITLE
Change GDI+ text

### DISCUSSION THREAD

**Discussion by jvolpin** [obs-websocket-community-projects/obs-websocket-java#152] [Q&A] [answered]
Hello, I wasn't able to find how to change GDI+ text, is this possible at the moment with the java library?

Thanks!

**Comment by ChristopheCVB** [comment, marked as answer]
Check the new protocol. The request you're after is [SetInputSettings](https://github.com/obsproject/obs-websocket/blob/master/docs/generated/protocol.md#setinputsettings) and the input settings field should be `text` if I remember correctly

**Comment by jvolpin** [comment]
Thank you very much!!
